scripts/2022_01_19_1_sip_10_update_pool_a_params.py

```diff
@@ -0,0 +1,37 @@
+from helpers import CHAIN_IDS, MULTISIG_ADDRESSES
+from ape_safe import ApeSafe
+from datetime import datetime, timedelta
+from brownie import accounts, network
+
+from scripts.utils import confirm_posting_transaction
+
+def main():
+    """Ramps A param for select pools as per SIP-10"""
+    print(f"You are using the '{network.show_active()}' network")
+    deployer = accounts.load("deployer")
+    multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS["MAINNET"]])
+
+    pools_to_future_a = {
+        "0xa6018520EAACC06C30fF2e1B3ee2c7c22e64196a": 1000, # SaddleALETHPool
```

Comment:
Our contract code doesn't allow us to move to an arbitrary A value, we can only double the current value at most in one update. Thus these values should be
aleth: 480
btcv2: 400
usdv2: 400

then we'll double them again in a couple weeks.

Reply:
btcv2 and usdcv2 are currently both at 400, so will be changing them both to 800

Reply:
please rm old file